File: app/src/main/java/com/the_roberto/kittengifs/giphy/RandomGifResponse.java
package com.the_roberto.kittengifs.giphy;

import com.google.gson.annotations.SerializedName;

public class RandomGifResponse {
    public Data data;

    public static class Data {
        @SerializedName("image_url") public String imageUrl;
        @SerializedName("image_mp4_url") public String imageUrlMp4;
    }
}

File: app/src/main/java/com/the_roberto/kittengifs/giphy/RandomGifResponse.kt
package com.the_roberto.kittengifs.giphy

import com.google.gson.annotations.SerializedName

class RandomGifResponse() {
    var data: Data? = null

    class Data {
        @SerializedName("image_url") var imageUrl: String? = null
        @SerializedName("image_mp4_url") var imageUrlMp4: String? = null
    }
}
